### PR TITLE
BaSP-TG-99: Implement shared component for Super Admin

### PR DESCRIPTION
**Pull request by CamilaFigue**
- Implement shared component for Super Admin (table, button, modal and logo).

- Add navigation from table to form on React Router Switch for Super Admin.

### CODE REVIEW

**Comment by cocolobos** [comment]
On Form component either add or edit, the Logo component is not being shown

File: src/Components/SharedComponents/Table/index.js [hunk 1]
```diff
@@ -32,13 +32,13 @@ const Table = ({ data, headers, children, handleEdit, deleteItem }) => {
...
                 <td>
-                  <Button type={styles.editANDdeleteBtn} handleClick={() => handleEdit(row._id)}>
+                  <Button class={styles.editANDdeleteBtn} handleClick={() => handleEdit(row._id)}>
                     Edit
                   </Button>
                 </td>
                 <td>
                   <Button
-                    type={styles.editANDdeleteBtn}
+                    class={styles.editANDdeleteBtn}
```
Comment (cocolobos) [inline src/Components/SharedComponents/Table/index.js:41]: we should rename this class

File: src/Components/SuperAdmins/SuperAdminForm.jsx [hunk 2]
```diff
@@ -90,11 +84,43 @@ const AddSuperAdmin = (props) => {
...
+          <p>Are you sure you want to delete this item?</p>
+          <p>You will not be able to recover it</p>
+        </div>
+        <div>
+          <Button class={styles.confirmANDdeleteBtn} handleClick={() => setIsOpen(false)}>
+            Cancel
+          </Button>
+          <Button
+            type="submit"
+            class={styles.confirmANDdeleteBtn}
```
Comment (cocolobos) [inline src/Components/SuperAdmins/SuperAdminForm.jsx:114]: we should rename this class

File: src/Components/SuperAdmins/index.js [hunk 3]
```diff
@@ -52,6 +53,9 @@ const SuperAdmin = (props) => {
   } else {
     return (
       <section className={styles.container}>
+        <div className={styles.logo}>
+          <Logo />
```
Comment (cocolobos) [inline src/Components/SuperAdmins/index.js:57]: This component is not align property on super admin table

File: src/Components/SuperAdmins/index.js [hunk 4]
```diff
@@ -52,6 +53,9 @@ const SuperAdmin = (props) => {
   } else {
     return (
       <section className={styles.container}>
+        <div className={styles.logo}>
```
Comment (Lucas-DeLorenzi) [inline src/Components/SuperAdmins/index.js:56]: maybe this div and its style property are not neccesary

File: src/Components/SuperAdmins/index.js [hunk 5]
```diff
@@ -52,6 +53,9 @@ const SuperAdmin = (props) => {
...
+          <Logo />
+        </div>
         <h2>Super Admin</h2>
```
Comment (Lucas-DeLorenzi) [inline src/Components/SuperAdmins/index.js:59]: the size of this title should be a bit bigger following figma design

File: src/Components/SuperAdmins/SuperAdminForm.jsx [hunk 6]
```diff
@@ -90,11 +84,43 @@ const AddSuperAdmin = (props) => {
...
         </Button>
       </form>
+      <Modal showModal={isOpen} closeModal={() => setIsOpen(false)}>
+        <h2>Warning</h2>
+        <div>
+          <p>Are you sure you want to delete this item?</p>
+          <p>You will not be able to recover it</p>
+        </div>
+        <div>
+          <Button class={styles.confirmANDdeleteBtn} handleClick={() => setIsOpen(false)}>
```
Comment (Lucas-DeLorenzi) [inline src/Components/SuperAdmins/SuperAdminForm.jsx:109]: this button should be of another color following figma design

File: src/Components/SuperAdmins/SuperAdminFormEdit.jsx [hunk 7]
```diff
@@ -105,11 +104,40 @@ const EditSuperAdmin = (props) => {
...
-        <button onClick={() => backSuperAdmin()}>Back</button>
+        </Button>
+        <Button class={styles.editANDdeleteBtn} handleClick={() => backSuperAdmin()}>
+          Back
+        </Button>
       </form>
+      <Modal showModal={isOpen} closeModal={() => setIsOpen(false)}>
+        <h2>Warning</h2>
+        <div>
+          <p>Are you sure you want to delete this item?</p>
```
Comment (Lucas-DeLorenzi) [inline src/Components/SuperAdmins/SuperAdminFormEdit.jsx:122]: Warning messages must be in accordance with the function being implemented

**Review by cocolobos — APPROVED** [review]

**Review by cocolobos — APPROVED** [review]

File: src/Components/SuperAdmins/super-admins.module.css [hunk 8]
```diff
@@ -1,22 +1,41 @@
...
   flex-direction: column;
+  justify-content: space-around;
+  margin: 30px;
 }
 
-.container h2 {
+.container > h2 {
   color: #373867;
   font-family: 'Arial';
   width: 150px;
```
Comment (cocolobos) [inline src/Components/SuperAdmins/super-admins.module.css:13]: I think we should make the width bigger so the title be in only one row

**Pull request merged**